### ISSUE TITLE
Google Analytics Support

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#7]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```

We have to think about the monetization strategy. Let's deal with Google Analytics support first since it's the easiest one to do.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```

Google Analytics support is available out of the box.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```

Google Analytics is not supported.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 0d1b32966dcd7313bf978de282a3969acaf66603

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
